fingertip_pressure/scripts/sim_sensor.py

```diff
@@ -57,20 +57,20 @@ def callback(self, pressurestate):
     def publish(self):
         ps = PressureState()
         ps.header.stamp = rospy.get_rostime();
-        ps.l_finger_tip = []
-        ps.r_finger_tip = []
```

Comment:
I don't see why this change would do anything?

Reply:
same as issue above, ensuring python3 is used seems to fix the issue, this can revert